vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -462,6 +481,48 @@ public void removeThemeNames(String... themeNames) {
         synchronizeTheme();
     }
 
+    /**
+     * Sets the current error message from the date time picker.
+     */
+    public void setErrorMessage(String errorMessage) {
```

Comment:
That was done in order to have similar docs for getter/setter. Practice that we are trying to follow in other pairs, so the difference is only `Gets`/`Sets`. Probably the wording can be updated for both.
Don't have strong opinion on this one, so updated.

Reply:
Ok. Now the wording on the getter sounds weird to me with "gets ... to". But at least it's understandable and not so important to keep tweaking. I think it's a bit artificially restrictive and might result in not so good docs in all cases if trying to keep the setter and getter docs so identical.

If doing that in this case though something like "which is displayed" instead of "to display" would work as well with either "gets" or "sets". But as I said it's not so important. I think it's good enough as it is. I'll mark this as resolved.